src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkTask.java

```diff
@@ -72,7 +81,8 @@ protected void start(Map<String, String> props, ElasticsearchClient client) {
 
     this.client = client != null ? client : new ElasticsearchClient(config, reporter);
 
-    log.info("Started ElasticsearchSinkTask.");
+    log.info("Started ElasticsearchSinkTask. Connecting to ES server version: {}",
```

Comment:
in the future scapping logs wont work. We should file another enhancement ticket to log this version as a metric this way we can plot it in datadog

Reply:
https://confluentinc.atlassian.net/browse/CCMSG-1218